cmd/web.go

```diff
@@ -145,7 +149,15 @@ func runWeb(ctx *cli.Context) error {
 
 	log.Info("Global init")
 	// Perform global initialization
-	routers.GlobalInit(graceful.GetManager().HammerContext())
+	setting.NewContextFromExistingConf()
+	routers.GlobalInitInstalled(graceful.GetManager().HammerContext())
+
+	// Now we can only check the AppDataPath here (the directory should already been created during installation)
+	// We can not check it in `GlobalInitInstalled`, because some integration tests
+	// use cmd -> GlobalInitInstalled, but the AppDataPath doesn't exist during the test.
```

Comment:
```suggestion
	// We check that AppDataPath exists here (it should have been created during installation)
	// We can't check it in `GlobalInitInstalled`, because some integration tests
	// use cmd -> GlobalInitInstalled, but the AppDataPath doesn't exist during those tests.
```

I guess we should fix those integration tests to have an extant AppDataPath - but probably not for this PR.